.github/workflows/cypress.yaml

```
@@ -81,58 +77,20 @@
-          parallel: true
-          group: "UI-Chrome"
-        env:
-          CYPRESS_SPLIT_TESTS: "true"
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NODE_OPTIONS: --max_old_space_size=4096
-          COMMIT_INFO_MESSAGE: ${{github.event.pull_request.title}}
-          COMMIT_INFO_SHA: ${{github.event.pull_request.head.sha}}
-          SPLIT: ${{ strategy.job-total }}
-          SPLIT_INDEX: ${{ strategy.job-index }}
+      - name: Run Cypress Tests in Parallel
+        run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2086:info:3:17: Double quote to prevent globbing and word splitting

(shellcheck)